src/components/quickstart/Mac.astro

```diff
@@ -48,12 +47,11 @@ const { latestVersion } = Astro.props
 
 <div class="prose dark:prose-invert">
   <p>
-    Launch Docker Desktop, accept the license agreement, and make sure Docker is
-    started and running.
+    Launch Orbstack, accept the license agreement.
```

Comment:
the only nitpick in the original there is the part about "and make sure docker is started and running" . not sure about the details of orbstack but is the vm started when orbstack is launched automatically or has the user still to start it after starting orbstack? in case that would apply to orbstack i would reintroduce the striked last part about started and running?

Reply:
aside that it looks good

Reply:
> but is the vm started when orbstack is launched automatically or has the user still to start it after starting orbstack

Yes, it is started automatically.